src/pages/project/project_wizard/RelatedRegistrationsField.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Typography } from '@mui/material';
+import { useFormikContext } from 'formik';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useFetchRegistrationsForProject } from '../../../api/hooks/useFetchRegistrationsForProject';
+import { ListPagination } from '../../../components/ListPagination';
+import { ListSkeleton } from '../../../components/ListSkeleton';
+import { RegistrationListItemContent } from '../../../components/RegistrationList';
+import { SearchListItem } from '../../../components/styled/Wrappers';
+import { CristinProject } from '../../../types/project.types';
+import { ROWS_PER_PAGE_OPTIONS } from '../../../utils/constants';
+
+export const RelatedRegistrationsField = () => {
+  const { t } = useTranslation();
+  const { values } = useFormikContext<CristinProject>();
+  const [page, setPage] = useState(1);
+  const [rowsPerPage, setRowsPerPage] = useState(ROWS_PER_PAGE_OPTIONS[0]);
+
+  const resultsQuery = useFetchRegistrationsForProject(values.id, rowsPerPage, page);
+  const results = resultsQuery.data;
+
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
```

Comment:
Kanskje synd å ha hele denne i en flex, da det fører til at det blir en ekstra space mellom søketreff og pagineringselementene som vi ikke har andre plasser?

![bilde](https://github.com/user-attachments/assets/e473084b-6c07-486f-9456-1ace9df9adc9)

Samme gjelder relaterte prosjekt også

Reply:
Tror det er her vi gjør det fordi vi trenger at <li> elementene er inni en <ul>

Reply:
Når jeg tenker meg om her, så burde vel egentlig `ListPagination` returnere en `<div>` i stedet for en `<>`, sånn at parent styling ikke gjør at paginering blir ulik på forskjellige plasser. Ser ikke at det fører til noe annet enn at paginering får ulik gaps. Men det er egentlig en annen sak.